app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: str) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
+
+    @classmethod
+    def get_person(cls, name: str) -> "Person":
+        return cls.people.get(name)
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person in people:
+        Person(person["name"], person["age"])
+    for person in people:
+        new_person = Person.get_person(person["name"])
+        if person.get("wife") is not None:
+            new_person.wife = Person.people[person["wife"]]
+            Person.people[person["wife"]].husband = new_person
+        if person.get("husband") is not None:
+            new_person.husband = Person.people[person["husband"]]
```

Comment:
will ur code work without this part ? u are adding husband for wife and wife for husband in code before